src/bin/cargo/commands/install.rs

```diff
@@ -16,6 +16,7 @@ use cargo_util::paths;
 pub fn cli() -> Command {
```

Comment:
In your PR description, could you compare this with #13598?

Also, could you explicitly call out in the description your stance on the discussion in #11123 about how much is being done during dry-run?

Reply:
I've updated the description as requested, does it suits you?

Reply:
Thanks!

Reply:
Sorry, I forgot something.  Once you've reslved y comments, if you could more explicitly call out what warnings are skipped and why, that'd help!

(probably should also say why in comments in the code)

Reply:
Finally with 86415010617b63c981fef8c18328331e11d47434, no warning are skipped.